firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -3493,6 +3522,35 @@ class BrowserViewController: UIViewController,
         }
     }
 
+    /// Configures the scrim area for zero search state
+    private func configureZeroSearchView() {
+        addressToolbarContainer.isHidden = false
+
+        zeroSearchDimmingView.alpha = 0
+        view.addSubview(zeroSearchDimmingView)
+        view.bringSubviewToFront(zeroSearchDimmingView)
+
+        NSLayoutConstraint.activate([
+            zeroSearchDimmingView.topAnchor.constraint(equalTo: contentContainer.topAnchor),
+            zeroSearchDimmingView.leadingAnchor.constraint(equalTo: contentContainer.leadingAnchor),
+            zeroSearchDimmingView.bottomAnchor.constraint(equalTo: contentContainer.bottomAnchor),
+            zeroSearchDimmingView.trailingAnchor.constraint(equalTo: contentContainer.trailingAnchor)
+        ])
+
+        UIView.animate(withDuration: 0.3) {
```

Comment:
Kept constant here instead of UX struct since it's easier to make changes here when doing design review. Happy to move it to struct in future if that's preferred.